### ISSUE TITLE
vggm not usable

### ISSUE THREAD

**Issue by noovd** [Cadene/pretrained-models.pytorch#52]
Hi,

currently `vggm` is not an usable architecture even though it is already implemented. Could you please fix it?

Thank you!

**Comment by Cadene** [comment]
@noovd I could not make vggm work in pytorch sorry. It uses old torch7 layers ported in pytorch (which does not work with cuda).

**Comment by noovd** [comment]
Could you be more precise as to what is broken? I am actually able to use your 'vggm' implementation with the given imagenet pretrained weights on gpu but the results I get are unexpectedly bad for several evaluation tasks eventhough i took normalization and RGB -> BGR into account. I was wondering if you could help me out there? Is this the part which is not working?

**Comment by samxuxiang** [comment]
@noovd 
Did you ensure that the input image range is in [0,255] before you apply normalization? 

I use the following code
transformations = transforms.Compose([
        transforms.Scale(224),
        transforms.ToTensor(),
        transforms.Lambda(lambda x: x.mul(255))
    ])
and then subtract the mean value

In my case I am able to fine-tune the model without any issue. Hope this help address your problem.

**Comment by ntuyt** [comment]
Do we need to convert the images from RGB -> BGR when using this VGGM model?

**Comment by Cadene** [comment]
@ntuyt Yes you do:
https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/models/vggm.py#L14

**Comment by ntuyt** [comment]
Hi, I have another question. Since input of original VGGM  224x224. But in your code, the input size is 221x221. Does it influence the performance. Meanwhile, I wonder the provided pretrained model is converted from which model?
